execution-service/src/main/java/org/ihtsdo/rvf/execution/service/WhitelistService.java

```diff
@@ -0,0 +1,30 @@
+package org.ihtsdo.rvf.execution.service;
+
+import org.ihtsdo.rvf.execution.service.whitelist.AcceptanceGatewayClient;
+import org.ihtsdo.rvf.execution.service.whitelist.AcceptanceGatewayClientFactory;
+import org.ihtsdo.rvf.execution.service.whitelist.WhitelistItem;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+import java.util.List;
+
+@Service
+public class WhitelistService {
+
+    @Autowired
+    private AcceptanceGatewayClientFactory factory;
+
+    public boolean isWhitelistDisabled() {
+        return factory.isWhitelistDisabled();
+    }
+
+    public  List<WhitelistItem> validateAssertions(List<WhitelistItem> assertions) {
```

Comment:
There doesn't seem to be any error handling here?
Please add a try block and throw some sort of service exception if the API call fails which has a message like `Acceptance Gateway communication failure`.
See https://www.baeldung.com/spring-rest-template-error-handling